src/content/1.7/development/webservice/_index.md

```diff
@@ -195,6 +195,61 @@ Here is what a resource API call could look like (in this case `http://example.c
 </prestashop>
 ```
 
+#### Available parameters
+
+You can add these **GET** parameters to your request to modify the READ response:
+
+* `display` to control which fields are returned
+* `filter` to control which items are returned
+* `language` to control which language values are returned
+
+##### Control returned fields with "display"
+
+The `display` parameter can be used to return all fields when used with the **full** value: `http://example.com/api/addresses/1?display=full`.
```

Comment:
The display parameter only works in listings, not for individual records. 

Examples:

* `http://example.com/api/addresses/` will display xlinks for all addresses
* `http://example.com/api/addresses/?display=full` will display all fields for all addresses
* `http://example.com/api/addresses/?display=id` will display ids for all addresses
* `http://example.com/api/addresses/?display=[id,id_supplier]` will display id and id_supplier for all addresses

I don't know if this is a bug or if it's intended, but using the `display` parameter for an individual record will return just the xlink, regardless of the value of `display`:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<prestashop xmlns:xlink="http://www.w3.org/1999/xlink">
    <address id="1" xlink:href="http://example.com/api/addresses/1"/>
</prestashop>
```

Reply:
Also there's also a magic value called `minimum` but I don't really know what it does. It's used by default so I guess it's id only.

Reply:
It seems to return only the primary key in the `id` field